frontend/views/utils/filters.js

```diff
@@ -12,14 +12,21 @@ export const getFileExtension = (
   return toUppercase ? ext.toUpperCase() : ext.toLowerCase()
 }
 
+const isPlayableAudioMime = (mimeType) => {
+  const audio = document.createElement('audio')
+  return !!audio.canPlayType(mimeType)
+}
```

Comment:
Could you add a comment above this function explaining why it's necessary at all? As far as I can tell, OGG's mimetype also begins with `audio/`... so what's this function for?